www/js/lib/zimArchive.js

```diff
@@ -501,6 +506,16 @@ ZIMArchive.prototype.findDirEntriesFromFullTextSearch = function (search, dirEnt
     });
 };
 
+ZIMArchive.prototype.getEntryDirByWasm = async function (path) {
```

Comment:
Elsewhere we unifromly use `getDirEntry`, not `getEntryDir`. It's confusing to have one function named differently. Also, as mentioned above, I don't think `ByWasm` is accurate, as we have WASMs of the other decompressors used as standard.

Reply:
My bad I left this here. I added this when I was prototyping